cwl/tools/anno_fusion.cwl

```diff
@@ -0,0 +1,25 @@
+class: CommandLineTool
+cwlVersion: v1.0
+id: format_fusion
+requirements:
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: 1000
+  - class: DockerRequirement
+    dockerPull: 'gaonkark/fusionanno:latest'
```

Comment:
We have fusionanno here: `docker pull pgc-images.sbgenomics.com/d3b-bixu/fusionanno:latest`, unless it is different, @kgaonkar6 ?

Reply:
Same here @zhangb1 @kgaonkar6

Reply:
ah yes using `pgc-images.sbgenomics.com/d3b-bixu/fusionanno:latest` will be better